test/unit/engine_target.js

```diff
@@ -323,6 +323,18 @@ test('duplicateVariable creates a new variable with a new ID by default', t => {
     t.end();
 });
 
+test('duplicateVariable creates new array reference for list variable.value', t => {
+    const target = new Target();
+    const arr = [1, 2, 3];
+    target.createVariable('a var ID', arr, Variable.LIST_TYPE);
```

Comment:
Hmmm not sure what's happening here but the second argument to `createVariable` should be a string not an array..

Reply:
```suggestion
    target.createVariable('a var ID', 'arr', Variable.LIST_TYPE);
```